dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/WCServerTest.java

```diff
@@ -390,7 +393,7 @@ public void testResponseReset() throws Exception {
      * @throws Exception
      */
     @Test
-    @SkipForRepeat("SERVLET-4.0")
+    @SkipForRepeat({"SERVLET-4.0", EE9_FEATURES})
```

Comment:
1) Do we even need the Servlet40 minor version test here? We already have a major/minor version test in the servlet 4.0 bucket I believe.
2) Lets explain why we skip the tests if that is the case.